Dockerfile

```diff
@@ -18,17 +19,24 @@ RUN /env/bin/pip install \
   rm -rf /tmp/dea_ml
```

Comment:
??

Reply:
This is from the original I didn't touch.